README.md

```diff
@@ -1 +1,65 @@
-# community
\ No newline at end of file
+# ROS 2 Security Working Group
+The Security Working Group's mission is to advocate for and implement security features within ROS 2.  The working group negotiates a balance between the benefits of security and the enablement of technology, striving for a secure-by-default design which can be customized to suit a targeted security profile.
+
+This document outlines the governance of the ROS 2 Security Working Group.  Updates to this document will be handled in the same manner as other project updates governed by the Working Group.
+
+## Subprojects
+A significant portion of the Security Working Group's mission is achieved by maintaining ROS subprojects.  The following subprojects are owned by this Working Group:
+ * _TBD:  list project names and team leads_
+
+### Out of Scope
+Although the following ROS subprojects have links to security, they are not governed by the Security Working Group:
+ * _TBD_
+
+### Adding and Maintaining Subprojects
```

Comment:
I think this should be split out. I like the tooling WG's use of issue templates for adding new projects.

Reply:
Added template.